kola/tests/misc/nfs.go

```diff
@@ -61,7 +55,23 @@ func NFS(c platform.TestCluster) error {
 		Hostname: "nfs1",
 	}
 
-	m1, err := c.NewMachine(c1.String())
+	mounttmpl = `[Unit]
+Description=NFS Client
+After=network-online.target
+Requires=network-online.target
+After=rpc-statd.service
+Requires=rpc-statd.service
+
+[Mount]
+What=%s:/tmp
+Where=/mnt
+Type=%s
+Options=defaults,noexec
```

Comment:
i'm not so sure - NFSv4 is failing against current alpha, but not against my local copy with updated nfs-utils. :confounded: